### DISCUSSION TITLE
Using latest Ocelot and MMLib.SwaggerForOcelot and Consul with dynamic routing

### DISCUSSION THREAD

**Discussion by elenacalugaru** [Burgyn/MMLib.SwaggerForOcelot#303] [General]
System.MissingMethodException: Method not found: 'System.Threading.Tasks.Task`1<System.Collections.Generic.List`1<Ocelot.Values.Service>> Ocelot.ServiceDiscovery.Providers.IServiceDiscoveryProvider.Get()'.
   at MMLib.SwaggerForOcelot.ServiceDiscovery.SwaggerServiceDiscoveryProvider.GetSwaggerUri(SwaggerEndPointConfig endPoint, RouteOptions route)

**Comment by raman-m** [comment]
# 🤣
